### PR TITLE
fix(odata-service-inquirer): #2307 Adds GA link for no services + error

### PR DESCRIPTION
**Pull request by IainSAP**
Fix for : #2307 

- Adds GA link for no services
- Adds test to cover

### CODE REVIEW

**Review by devinea — APPROVED** [review]
Code changes are clear and covered by tests.
Changeset 👍

**Review by longieirl — APPROVED** [review]
Changes look good.
Test coverage is good.
Changeset present.

File: packages/odata-service-inquirer/src/prompts/datasources/sap-system/new-system/service-helper.ts [hunk 1]
```diff
@@ -58,26 +60,53 @@ const createServiceChoices = (serviceInfos?: ODataServiceInfo[]): ListChoiceOpti
...
+/**
+ * Logs the catalog reuest errors.
+ *
+ * @param requestErrors catalog request errors
+ */
+function logErrorsForHelp(requestErrors: Record<ODataVersion, unknown> | {}): void {
+    // Log the first error only
+    const catalogErrors = Object.values(requestErrors);
+    if (catalogErrors.length > 0) {
+        forEach(catalogErrors, (error) => {
```
Comment (cianmSAP) [inline packages/odata-service-inquirer/src/prompts/datasources/sap-system/new-system/service-helper.ts:72]: Just a query, why the use of `lodash` forEach here?

File: packages/odata-service-inquirer/test/unit/prompts/cap-project/questions.test.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import * as capValidators from '../../../../src/prompts/datasources/cap-project/
 import type { CapCustomPaths } from '@sap-ux/project-access';
 import { errorHandler } from '../../../../src/prompts/prompt-helpers';
 import { type CdsVersionInfo } from '@sap-ux/project-access';
+import { error } from 'console';
```
[the remark this answers is not in the document]
Comment (cianmSAP) [reply packages/odata-service-inquirer/test/unit/prompts/cap-project/questions.test.ts:16]: Can be removed in a follow up

[the remark this answers is not in the document]
Comment (IainSAP) [reply packages/odata-service-inquirer/test/unit/prompts/cap-project/questions.test.ts:16]: OK, will do Im sure it wont be long before Im updating these anyway. Thx

File: packages/odata-service-inquirer/src/prompts/datasources/sap-system/new-system/service-helper.ts (hunk 1 above)
Comment (IainSAP) [reply packages/odata-service-inquirer/src/prompts/datasources/sap-system/new-system/service-helper.ts:72]: Auto-import 😮‍💨

**Review by cianmSAP — APPROVED** [review]
Comments addressed
Code updates look good
Changeset ✅

**Review by devinea — APPROVED** [review]
Re-approving

**Pull request merged**